### DISCUSSION TITLE
Export log - any way to access programmatically?

### DISCUSSION THREAD

**Discussion by jcolinger3** [InternetConnectionMonitor/icm-public#122] [Ideas and Feature requests]
I'm looking to collect a bunch of logs for analysis.  I've found the extension folder and the log.html.  I should be able to collect that.  But it isn't the best to work with.

Can there be a log.csv that is available for collection?  Or could you add an (optional) automated way to deliver the log to a destination (sftp/Google Drive)?

Or does anyone know of a solution similar to ICM that allows for similar testing/logging that can be collected?

**Comment by dmishh** [comment]
@jcolinger3 at this moment, ICM cannot automatically export the log to sFTP, Google Drive, email, etc. Here is a feature request - #84 

However, you can manually download the log in CSV format and tune a variety of export options. Open **Connection log** from the main popup and press Export to CSV

<img width="500" alt="image" src="https://github.com/user-attachments/assets/5639f3c9-0ba6-4b37-ad80-58a261b07818">
